### DISCUSSION TITLE
Add a user label for every RFKey

### DISCUSSION THREAD

**Discussion by cnico** [arendst/Tasmota#17715] [Ideas]
Hi, 

I use tasmota on a sonoff rf bridge and it works very well for years now.
I wanted to add a new rf command and I realized I did not know anymore to what each RFKey command is related in order not to reset a useful command. After a few test, I found it back :).

But I thought, it would be nice if you could add a label to the rfkey in order to describe it and not have to remember what each number is for. It could be short if memory is a problem :  10 or 20 caracters should be enough. And it should be displayed in the tasmota interface in the same box as the button number.
Another small improvement could to have a different typo for a RFKey button that has a default signal and one that has a user signal (for exemple a bold outline). So that we simply know which RFKeys are available and which are not.

If you could add this to tasmota, it would be nice.

Thanks,

**Comment by barbudor** [comment]
There is no default key settings.
By default, every Key is empty

I don't think we would waste 16 x 10 = 160 bytes for that, and 320 bytes even less.

If you don't use all rules, you can store your memo in a rule that remains OFF
Otherwise, a text file on your PC, or tools like OneNote, Notion, etc ... are the best to keep details of your infrastructure

**Comment by Th0maz** [comment]
You can do that already with `Webbutton<x>`.
I just tried:
`Webbutton1 test`

![image](https://user-images.githubusercontent.com/12065628/213481161-8d511a5f-c08b-4ccd-b724-263330050f09.png)

**Reply by cnico** [reply]
It works perfectly well. 
Thanks !
